### ISSUE TITLE
Support for external XJT Module on Jumper T20

### ISSUE THREAD

**Issue by dadantsou** [EdgeTX/edgetx#4713]
### Is there an existing issue for this feature request?

- [X] I have searched the existing issues

### Is your feature request related to a problem?

There is no support for XJT when selecting protocol for external module on Jumper T20

### Describe the solution you'd like

I wish an implementation that can activate XJT protocols for external module

**Comment by pfeerick** [comment]
XJT or XJT Lite?

**Comment by dadantsou** [comment]
XJT

**Comment by pfeerick** [comment]
Out of curiosity, how are you attaching the JR Micro XJT to the T20? Custom adapter? As I thought there was only a Nano form factor module bay available for it?

**Comment by 3djc** [comment]
Wouldn't a nano multi make a much better job without the utter ugliness of a JR module on T20 ?

**Comment by dadantsou** [comment]
I don't want to buy another XJT module, I want just use it for my old Frsky receiver.
If possible, I will design later a slimmer case for it

**Comment by pfeerick** [comment]
Hm... interesting. 

Custom firmware with `MODULE_SIZE_STD` instead of `MODULE_SIZE_SML` maybe?

**Comment by 3djc** [comment]
yes; was going to suggest he forks and make custom firmware with that change, understanding that it closes the door to other modules

**Comment by dadantsou** [comment]
OK, thanks guys

**Comment by pfeerick** [comment]
Do you have a link to that adapter btw? Looks handy for some Frankenstein testing purposes :zany_face:

**Comment by dadantsou** [comment]
I've found it on Aliexpress.
[https://www.aliexpress.com/item/1005004743012876.html?spm=a2g0o.cart.0.0.4b9238dagJDQSu&mp=1](url)

**Comment by pfeerick** [comment]
Thanks for that... I was hoping it was going to be something could 3d print myself, but still might be handy. 

@3djc What do you think about an option in the radio settings that makes it possible for all RF module types the radio can actually work with selectable? Otherwise, I think we can safely leave this as a "too much of an edge case" scenario, with a custom firmware build being most appropriate.
